website/content/docs/developers/compiling/compiling.md

```diff
@@ -100,7 +107,7 @@ $ bazel build --config=PLATFORM scripts/packages:tarpkgs
 
 Resulting artifacts can be found in subdirectories below the `bazel-bin`
 directory. The `heron-tracker` executable, for example, can be found at
-`bazel-bin/heron/tracker/src/python/heron-tracker`.
+`bazel-bin/heron/tools/tracker/src/python/heron-tracker`.
```

Comment:
Looks like there are a couple more instances of `heron/tracker`, as well as `heron/cli` and `heron/ui` in the docs. Would you mind please fixes those as well?